charts/k8s-monitoring/charts/feature-application-observability/README.md

```diff
@@ -40,6 +40,41 @@ Be sure perform actual integration testing in a live environment in the main [k8
 
 ## Values
 
+### Connectors: Grafana Cloud Host Info
+
+| Key | Type | Default | Description |
+|-----|------|---------|-------------|
+| connectors.grafanaCloudMetrics.enabled | bool | `true` | Generate host info metrics from telemetry data, used in Application Observability in Grafana Cloud. |
```

Comment:
```suggestion
| connectors.grafanaCloudMetrics.enabled | bool | `true` | Generate host info metrics from telemetry data, used in Application Observability in Grafana Cloud. DO NOT enable this if you do not wish to use Application Observability. |
```

Reply:
If customers use this feature w/o intended to use (and pay) for App O11y, having this enabled will result in billing calculations to be done for App O11y host hour based pricing. We should probably add a warning here so users aren't surprised when that happens.